package.json

```diff
@@ -102,20 +108,39 @@
       "**/*.(t|j)s"
     ],
     "coverageDirectory": "../coverage",
-    "testEnvironment": "node"
+    "testEnvironment": "node",
+    "moduleNameMapper": {
+      "@modules(.*)": "<rootDir>/dist/modules/$1",
+      "@providers(.*)": "<rootDir>/dist/providers/$1",
+      "@constants(.*)": "<rootDir>/dist/constants/$1",
+      "@config(.*)": "<rootDir>/dist/config/$1"
+    }
   },
   "_moduleAliases": {
     "@filters": "./dist/filters",
     "@guards": "./dist/guards",
     "@pipes": "./dist/pipes",
     "@interceptors": "./dist/interceptors",
-    "@constants": "./dist/constants",
     "@interfaces": "./dist/interfaces",
     "@decorators": "./dist/decorators",
     "@modules": "./dist/modules",
     "@config": "./dist/config",
     "@providers": "./dist/providers",
     "@helpers": "./dist/helpers",
-    "@types": "./dist/types"
+    "@types": "./dist/types",
+    "@constants": "./dist/constants"
+  },
+  "config": {
+    "commitizen": {
+      "path": "node_modules/cz-customizable"
+    },
+    "cz-customizable": {
+      "config": "commitizen.js"
+    }
+  },
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS -g 'commitlint.js'"
```

Comment:
if you tries add husky, please create it at another PR and do it clear. Create .husky folder etc